.github/actions/get-python-version.yaml

```diff
@@ -0,0 +1,14 @@
+name: "Get Python Version"
+description: "Read the python version from .python-version file"
+outputs:
+  python-version:
+    description: "Python version"
+    value: ${{ steps.get_version.outputs.python-version }}
+runs:
+  using: "composite"
+  steps:
+    - name: get python-version
+      shell: bash
+      id: get_version
+      run: |
+        echo "python-version=$(cat .python-version)" >> $GITHUB_OUTPUT
```

Comment:
there is no .python-version file in the project, wont this fail and break the build?

Reply:
[fixed](https://github.com/robusta-dev/krr/pull/388/commits/36f95764f90e9556c083ab33ba1709c141b5a602)